### PR TITLE
Optimize english documentation

### PR DESCRIPTION
**Pull request by shadowlion**
I was reading the Nuxt.js documentation under the "Guides" section, thought there were a few fixes I could help with:

**Installation.md**

- one of the `create-nuxt-app` commands was misspelled.
- changed some wording to have more actionable verbiage
- changed wording so it was editor-agnostic; I think beginners will choose whatever editor they want to choose, so rather, the code  should reflect the inherent idea behind the example.

**Routing.md**

- used Vetur practices on some of the code examples
- `<NuxtLink>` can also be converted to `<nuxt-link>`

**Conclusion.md**

- made sentencing more actionable (i.e. less passive tense)

### CODE REVIEW

File: content/en/guides/get-started/routing.md [hunk 1]
```diff
@@ -36,20 +42,20 @@ The `<NuxtLink>` component should be used for all internal links. That means for
...
+      About (internal link that belongs to the Nuxt App)
+    </NuxtLink>
     <a href="https://nuxtjs.org">External Link to another page</a>
   </main>
 </template>
 ```
 
 <app-modal>
-  <code-sandbox  :src="csb_link"></code-sandbox>
+  <code-sandbox :src="csb_link" />
```
Comment (debs-obrien) [inline content/en/guides/get-started/routing.md:54]: You cannot use self-closing tags, i.e., this won't work in content module
https://content.nuxtjs.org/writing#vue-components

File: content/en/guides/get-started/routing.md [hunk 2]
```diff
@@ -20,7 +20,13 @@ Learn more about [Routing](/guides/features/file-system-routing)
 
 ## Navigation
 
-To navigate between pages of your app, you should use the [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component) component. This component is included with Nuxt.js and therefore you don't have to import it like you do with other components. It is similar to the HTML `<a>` tag except that instead of using a `href="/about"` we use `to="/about"`. If you have used `vue-router` before, you can think of the `<NuxtLink>` as a replacement for `<RouterLink>`
+To navigate between pages of your app, you should use the [NuxtLink](/guides/features/nuxt-components#the-nuxtlink-component) component. This component is included with Nuxt.js and therefore you don't have to import it like you do with other components. It is similar to the HTML `<a>` tag, except that instead of using a `href="/about"` we use `to="/about"`. If you have used `vue-router` before, you can think of the `<NuxtLink>` as a replacement for `<RouterLink>`
+
+<base-alert type="info">
+
+Keep in mind, as you insert `<NuxtLink>` in your template, you may spell it out either as `<NuxtLink>` or `<nuxt-link>`
```
Comment (debs-obrien) [inline content/en/guides/get-started/routing.md:27]: Although you can use nuxt-link we are advising to use NuxtLink as a standard therefore do not want to add  nuxt-link here at least at beginner stage and not as a suggestion.

**Comment by debs-obrien** [comment]
Nice work @shadowlion. just some minor issues to resolve.

File: content/en/guides/get-started/routing.md (hunk 1 above)
Comment (shadowlion) [reply content/en/guides/get-started/routing.md]: Got it, fixed - 4bd375a

File: content/en/guides/get-started/routing.md (hunk 2 above)
Comment (shadowlion) [reply content/en/guides/get-started/routing.md]: Ah, I see - fixed - 4bd375a

**Pull request merged**